app/views/declaration/declaration_holder.scala.html

```diff
@@ -20,62 +20,121 @@
 
-@main_template(
-    title = Title("supplementary.declarationHolder.title", "supplementary.summary.parties.header")
-) {
+@addButton = @{
+govukButton(Button(
+    classes = "govuk-button--secondary",
+    content = HtmlContent(HtmlFormat.fill(
+        immutable.Seq(
+            Html(s"""<span aria-hidden="true">${messages("site.add")}</span>"""),
+            Html(s"""<span class="govuk-visually-hidden">${messages("supplementary.additionalActors.add.hint")}</span>""")
+        )
+    )),
+    name = Some("Add"),
+    attributes = Map("id" -> "add")
+))
+}
 
-    @components.back_link(Navigator.backLink(DeclarationHolder, mode), mode)
+@removeButton(value: Option[String]) = @{
+govukButton(Button(
+    classes = "govuk-button--secondary govuk-!-margin-0",
+    content = HtmlContent(HtmlFormat.fill(
+        immutable.Seq(
+            Html(s"""<span aria-hidden="true">${messages("site.remove")}</span>"""),
+            Html(s"""<span class="govuk-visually-hidden">${messages("site.remove.hint", value.getOrElse(""))}</span>""")
+        )
+    )),
+    name = Some(messages("site.remove")),
+    value = value
+))
+}
 
-    @components.error_summary(form.errors)
+@holdersTable = {
+    @if(holders.nonEmpty) {
+        @govukTable(Table(
+            rows = holders.map(_.toString).zipWithIndex.map { case (holder, index) =>
+                Seq(
+                    TableRow(
+                        content = Text(holder),
+                        attributes = Map("id" -> s"holder_$index")
+                    ),
+                    TableRow(
+                        content = HtmlContent(removeButton(Some(holder))),
+                        attributes = Map("id" -> s"holder_action_$index")
+                    )
+                )
+            },
+            head = Some(List(
+                HeadCell(
+                    content = Text(messages("supplementary.declarationHolder.authorisationCode")),
+                    colspan = Some(2)
+                )
+            ))
+        ))
+    }
+}
+
+@govukLayout(
+    title = Title("supplementary.declarationHolder.title", "supplementary.summary.parties.header"),
+    backButton = Some(BackButton(messages("site.back"), Navigator.backLink(DeclarationHolder, mode)))
+) {
 
-    @components.heading(
-        title = messages("supplementary.declarationHolder.title"),
-        sectionHeader = messages("supplementary.summary.parties.header")
-    )
+    @formHelper(action = DeclarationHolderController.submitHoldersOfAuthorisation(mode), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
 
-    <p>@messages("supplementary.declarationHolder.title.hint")</p>
+        @sectionHeader(messages("supplementary.summary.parties.header"))
 
-    @helper.form(DeclarationHolderController.submitHoldersOfAuthorisation(mode), 'autoComplete -> "off") {
-        @helper.CSRF.formField
+        @pageTitle(messages("supplementary.declarationHolder.title"))
```

Comment:
Can we take this opportunity to rename `supplementary.declarationHolder...` messages to `declaration.declarationHolder...`